docs/available_commands.md

```diff
@@ -164,31 +277,37 @@ So only a message that the component was not found and could not be updated is l
 
 ## validate
 
-This command is used to validate the SBOM according to a specification.
+This command is used to validate the SBOM against a JSON schema.
 
-### Use of different schemas
+### Schema selection
 
-The package is currently delivered with the specification for CycloneDX 1.3 and 1.4. Further, it is provided with a custom schema, which not only requires the minimum elements as defined by the [NTIA](https://www.ntia.gov/files/ntia/publications/sbom_minimum_elements_report.pdf) but also some further recommended fields, e.g. licenses and stating the known unknowns (through the `compositions`-field).
-You can control the usage of the specification with the flag `--schema-type`:
+This tool can validate SBOMs against any user-provided JSON schema but for convenience, two schema types are built in:
 
-    cdx-ev validate bom.json --schema-type=custom # use provided custom schema in package
-    cdx-ev validate bom.json # default CycloneDX specification will be used
+* The *default* schema type validates against the [stock CycloneDX schema](https://github.com/CycloneDX/specification).
+* The *custom* schema type uses a more restrictive schema which accepts a subset of CycloneDX. Additional requirements incorporated into the schema mostly originate from the [NTIA](https://www.ntia.gov/files/ntia/publications/sbom_minimum_elements_report.pdf) but also include some of the authors' recommendations.
```

Comment:
Not completely correct, even there we have an overlap with chapter 5 "Beyond Minimum Elements: Enabling Broader SBOM Use Cases", e.g. licenses.

Reply:
Can you be more precise or even suggest another wording?
As it is, I don't know which part of what I wrote is inaccurate.

Reply:
I was just talking about the last part "some of the authors' recommendations". Though it is right that it's our scheme, the NTIA also recommends having this field (see chapter 5 in the linked document)

Reply:
In the latest change, I've simply removed the last half sentence. That way, the meaning should be sufficiently vague to also encompass the truth 😉 

If you want to be more precise about where stuff comes from, I'd kindly ask you to suggest a correct wording.